public/includes/header.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// error_reporting(0);
```

Comment:
What is the purpose of this line, please? Can we remove it if is not needed in production, I don't think it is a good idea to disable error reporting in production.